### PR TITLE
HYDRA-1043 : Fix Bifrost graphs can't be displayed in Hydra viewport

### PR DESCRIPTION
**Pull request by lanierd-adsk**

### CODE REVIEW

**Comment by lanierd-adsk** [comment]
FYI : @ppt-adsk @debloip-adsk

**Comment by lanierd-adsk** [comment]
BTW: no unit test is added as we don't support Bifrost yet in automation, but a ticket is logged for that.

**Comment by lanierd-adsk** [comment]
Locally the the tests pass. I don't know why I get segment faults and random failures with the tests. Giving it another try.

File: test/lib/mayaUsd/render/mayaToHydra/testMayaDefaultMaterial.py [hunk 1]
```diff
@@ -45,15 +46,14 @@ def test_MayaDefaultMaterial(self):
             cmds.refresh()
             self.assertSnapshotClose("defaultMaterial" + ".png", self.IMAGE_DIFF_FAIL_THRESHOLD, self.IMAGE_DIFF_FAIL_PERCENT)        
 
+    @unittest.skipUnless(platform.system() != "Darwin", 'This test is disabled on OSX as not all implicit surfaces are fully implemented on OSX.')
```
Comment (lilike-adsk) [inline test/lib/mayaUsd/render/mayaToHydra/testMayaDefaultMaterial.py:49]: Who will support them on OSX? If it's us, should we track via a ticket?

Comment (lanierd-adsk) [reply test/lib/mayaUsd/render/mayaToHydra/testMayaDefaultMaterial.py:49]: I don't think we should take the ownership of those on OSX.
We should see if a ticket is logged in OpenUSD about that and if not log one.

Comment (lanierd-adsk) [reply test/lib/mayaUsd/render/mayaToHydra/testMayaDefaultMaterial.py:49]: I created https://github.com/PixarAnimationStudios/OpenUSD/issues/3305 in the OpenUSd github to see if there is already something logged.

**Review by lilike-adsk — APPROVED** [review]

**Pull request merged**